CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -397,12 +394,11 @@ local function MakeButton(name, text, size, clickFunc, pageRef, hubRef)
 	}
 
 	if clickFunc then
-		button.MouseButton1Click:connect(function()
+		button.MouseButton1Click:Connect(function()
 			local lastInputType = nil
-			pcall(function() lastInputType = UserInputService:GetLastInputType() end)
-			if lastInputType then
-				clickFunc(lastInputTypee == Enum.UserInputType.Gamepad1 or lastInputType == Enum.UserInputType.Gamepad2 or
-					lastInputType == Enum.UserInputType.Gamepad3 or lastInputType == Enum.UserInputType.Gamepad4)
+			local success, lastInputType = pcall(UserInputService.GetLastInputType, UserInputService)
```

Comment:
we can in general remove the pcall here, it only existed because we potentially shipped this code before the api for lastinputtype existed